itenium-socks/src/app/pipes/price.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'price',
+  standalone: true
+})
+export class PricePipe implements PipeTransform {
+
+  transform(value: unknown, ...args: unknown[]): unknown {
```

Comment:
Als je overal `unknown` gebruikt kan je evengoed geen TypeScript gebruiken....

```
transform(value: number): string { }
```

Als je dan doet:

```
{{ 'string' | price }}
```

Dan krijg je:
![image](https://github.com/itenium-be/Angular-Days/assets/2582910/6577d971-b89e-40a8-97ce-dbef44b4b93a)